assets/app/widgets/crowbar-checklist/crowbar-checklist.directive.spec.js

```diff
@@ -0,0 +1,144 @@
+/*global bard $httpBackend assert $rootScope $compile */
+describe('Crowbar Checklist Directive', function () {
+
+    var directiveElement,
+        scope;
+
+    beforeEach(function () {
+        bard.appModule('crowbarApp');
+
+        bard.inject('$rootScope', '$compile', '$httpBackend');
+
+        //Mock requests that are expected to be made
+        $httpBackend.expectGET('app/features/upgrade/i18n/en.json').respond({});
+        $httpBackend.flush();
+
+        scope = $rootScope.$new();
+
+        directiveElement = compileDirective(
+            '<crowbar-checklist checklist="testChecks" completed="testCompleted"></crowbar-checklist>'
+        );
+    });
+
+    function compileDirective(template) {
+        var element = angular.element(template),
+            compiledElement = $compile(element)(scope);
+        scope.$digest();
+
+        return compiledElement;
+    }
+
+    // Verify no unexpected http call has been made
+    bard.verifyNoOutstandingHttpRequests();
+
+    describe('with empty checks list', function () {
+        beforeEach(function () {
+            scope.testChecks = {};
+            scope.$digest();
+        });
+
+        it('should contain empty UL', function () {
+            var uls = directiveElement.find('ul');
+            expect(uls.length).toEqual(1);
+            expect(angular.element(uls[0]).children().length).toEqual(0);
+        });
+    });
+
+    describe('with non-empty checks list', function () {
+        beforeEach(function () {
+            scope.testChecks = {
+                check1: { status: false, label: 'label1' },
+                check2: { status: true, label: 'label2' },
+                check3: { status: false, label: 'label3' }
+            };
+            scope.$digest();
+        });
+
+        it('should contain a list with one item for each check', function () {
+            var uls = directiveElement.find('ul');
+            expect(uls.length).toEqual(1);
+            expect(angular.element(uls[0]).children().length).toEqual(_.size(scope.testChecks));
+        });
+
+        it('should have all items with labels maching the model', function () {
+            var lis = directiveElement.find('li');
+            _.each(lis, function (li) {
+                var itemElement = angular.element(li)
```

Comment:
Missing semicolon.
Also, please use one `var` statement to declare multiple variables